analytics/consent_test.go

```diff
@@ -49,13 +49,13 @@ func TestConsentPromptsForConsentIfConsentNotRecorded(t *testing.T) {
 	consentPath = newConsentTempfile(t)
 
 	var outbuf bytes.Buffer
-	out = &outbuf
+	out := &outbuf
 	var inbuf bytes.Buffer
 	inbuf.Write([]byte("\n"))
-	in = &inbuf
+	in := &inbuf
```

Comment:
Do you really need these two variables (`in` and `out`)?

Reply:
@pavelnikolov good call – I have refactored them out in 45d1c59.